CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,296 @@
+* Privileges:
+    * Invitations to contributor events
+    * Eligible to become an Organization Member
+
+
+### Organization Member
+
+An Organization Member is an established contributor who regularly participates in the project. Organization Members have privileges in both project repositories and elections, and as such are expected to act in the interests of the whole project.
+
+An Organization Member must meet the responsibilities and has the requirements of a Contributor, plus:
+
+* Responsibilities include:
+   * Continues to contribute regularly
+   * Help uphold our commmunity values and welcome newcomers
+* Requirements:
+   * Enabled [two-factor
+     authentication](https://help.github.com/articles/about-two-factor-authentication)
+     on their GitHub account
+   * Have made multiple contributions to the project or community. Contributions
+     may include, but is not limited to:
+      * Authoring or reviewing PRs on GitHub
+      * Filing or commenting on issues on GitHub
+      * Contributing to subprojects, or community discussions (e.g. meetings,
+        chat, email, and discussion forums)
+      * [Joined the Slack channel](https://cloud-native.slack.com/archives/C0344AANLA1)
+         * [Get an invite to join CNCF](http://slack.cncf.io/)
+      * Have read the [contributor
+        guide](https://github.com/open-feature/.github/blob/main/CONTRIBUTING.md)
+      * Actively contributing to 1 or more subprojects.
+* Privileges:
+    * May be assigned Issues and Reviews
+    * May give commands to CI/CD automation
+    * Entitled to vote in the [TODO: appropriate election]
+    * Can be added to [TODO: Repo Host] teams
+    * Can recommend other contributors to become Org Members
+    
+The process for a Contributor to become an Organization Member is as follows:
+
+ 1. Sponsored by 2 approvers. Note the following requirements for sponsors:
+         * Sponsors must have close interactions with the prospective member - e.g.
+           code/design/proposal review, coordinating on issues, etc.
+         * Sponsors must be approvers or maintainers in at least 1 CODEOWNERS file
+           in any repo in the OpenFeature org.
+         * Sponsors must be from multiple member companies to demonstrate integration
+           across community.
+ 2. [Open an issue](https://github.com/open-feature/community/issues/new)
+   * Ensure your sponsors are `@mentioned` on the issue
+   * Complete every item on the checklist ([preview the current version of the
+        template](https://github.com/open-feature/community/ISSUE_TEMPLATE/membership.md))
+   * Make sure that the list of contributions included is representative of your
+        work on the project.
+ 3. Have your sponsoring reviewers reply confirmation of sponsorship: `I support`
+ 4. Once your sponsors have responded, your request will be reviewed by the
+   Technical Committee (TC).  Any TC member can review the requirements and add
+   Members to the GitHub org.
```

Comment:
I think we need to have the bootstrap TC members approve this PR eventually, since they are implicated here.

Reply:
I agree, I also think we should get this out there and make changes as we go